pkg/apis/node/v1alpha1/conversion.go

```diff
@@ -33,11 +33,17 @@ func addConversionFuncs(s *runtime.Scheme) error {
 func Convert_v1alpha1_RuntimeClass_To_node_RuntimeClass(in *v1alpha1.RuntimeClass, out *node.RuntimeClass, s conversion.Scope) error {
 	out.ObjectMeta = in.ObjectMeta
 	out.Handler = in.Spec.RuntimeHandler
+	if in.Spec.Topology != nil {
+		autoConvert_v1alpha1_Topology_To_node_Topology(in.Spec.Topology,out.Topology,s)
```

Comment:
oh, you probably need to allocate an output topology for this to work
```
out.Topology = new(node.Topology)
```

Reply:
oh yes, thanks !